scripts/variantstore/docs/gvs-bulk-ingest-details.md

```diff
@@ -0,0 +1,55 @@
+# Bulk Ingest Details
+
+
+To enable the ingestion of >10k samples at a time, we now support bulk ingestion. It is most straightforward to use the default instructions in [Run Your Own Samples](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/beta_docs/run-your-own-samples.md). However, if your data doesn't follow the recommended table structure, this document outlines the optional parameters you can use to use GVS.
+
+---
+
+#### Input descriptions
+
+The table below describes the GVS ingest variables:
+
+| Input variable name         | Description                                                                                                                                                                | Type |
+|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------| --- |
+| samples_table_name          | (Optional) The name of the data table; This table holds the GVCFs to be ingested; `sample` is the default.                                                                 | String |
+| sample_id_column_name       | (Optional) User defined sample id column name; `sample_id` is the recommended name;                                                                                        | String |
+| vcf_files_column_name       | (Optional) Column that contains the GCS paths to the sample GVCF files. Determined by file inspection if not supplied.                                                     | String |
+| vcf_index_files_column_name | (Optional) Column that contains the GCS paths to the sample GVCF index files. Determined by file inspection if not supplied.                                               | String |
+| sample_set_name             | (Optional) The name of the set of samples. Only required when doing subsets of data in the `sample` table.                                                                 | String |
```

Comment:
```suggestion
| sample_set_name             | (Optional) The name of the set of samples. Only required when doing subsets of data in the `sample` table; otherwise all the samples will be ingested.                                                                 | String |
```